test/BaselineOutput/SingleDebug/Command/Datatypes-datatypes.txt

```diff
@@ -0,0 +1,18 @@
+#@ TextLoader{
+#@   header+
+#@   sep=tab
+#@   col=bl:BL:0
+#@   col=i1:I1:1
+#@   col=i2:I2:2
+#@   col=i4:I4:3
+#@   col=i8:I8:4
+#@   col=ts:TS:5
+#@   col=dto:DZ:6
+#@   col=dt:DT:7
+#@   col=tx:TX:8
+#@ }
+bl	i1	i2	i4	i8	ts	dto	dt	tx
+0	127	32767	2147483647	9223372036854775807	"2.00:00:00"	"2008-11-30T00:00:00.0000000+00:00"	"2013-08-05T00:00:00.0000000"	foo
+1	-127	-32767	-2147483647	-9223372036854775807	"7.00:00:00"	"2008-11-30T00:00:00.0000000+00:00"	"2013-08-05T00:00:00.0000000"	xyz
+					"7.00:00:00"	"2008-11-30T00:00:00.0000000+00:00"	"2013-08-05T00:00:00.0000000"	xyz
```

Comment:
>xyz [](start = 84, length = 3)

Maybe this one ought to be empty for text, since that is an interesting boundary case as well.

Reply:
@tfinley@gmail.com The line below contains empty string. 

---
In reply to: [213097496](https://github.com/dotnet/machinelearning/pull/750#discussion_r213097496) [](ancestors = 213097496)

Reply:
No, it does not. That is a *missing* string. Under default settings of the text loader and saver, the text field that has no content is considered missing, but with two quotes is considered the empty string. `""` vs. nothing.